Client/Frontend/Toolbar+URLBar/TabToolbarHelper.swift

```diff
@@ -90,14 +99,22 @@ open class TabToolbarHelper: NSObject {
     // Default state as reload
     var middleButtonState: MiddleButtonState = .home
 
+    private var longPressGestureRecognizers: [UILongPressGestureRecognizer] = []
+    private let uiLargeContentViewInteraction: UILargeContentViewerInteraction = .init()
+
     init(toolbar: TabToolbarProtocol) {
         self.toolbar = toolbar
         super.init()
 
+        toolbar.addUILargeContentViewInteraction(interaction: uiLargeContentViewInteraction)
+
         toolbar.backButton.setImage(UIImage.templateImageNamed(StandardImageIdentifiers.Large.back)?.imageFlippedForRightToLeftLayoutDirection(), for: .normal)
         toolbar.backButton.accessibilityLabel = .TabToolbarBackAccessibilityLabel
         toolbar.backButton.accessibilityIdentifier = AccessibilityIdentifiers.Toolbar.backButton
+        toolbar.backButton.showsLargeContentViewer = true
+        toolbar.backButton.largeContentTitle = .TabToolbarBackAccessibilityLabel
```

Comment:
I reused an existing accessibility label. Should I create a separate localized string for the `largeContentTitle`?